alvr/audio/src/lib.rs

```diff
@@ -30,6 +30,7 @@ use std::{
 
 static VIRTUAL_MICROPHONE_PAIRS: Lazy<HashMap<&str, &str>> = Lazy::new(|| {
     [
+        ("Virtual Cable 1", "Virtual Cable 2"),
         ("CABLE Input", "CABLE Output"),
```

Comment:
Since you've already gone ahead and put the change in the wiki, you should also update the VB-Cable names here

Reply:
I recall an issue about some device name being changed, but I can't find it anymore. Maybe I'm also confusing it with the feature request for VAC

Reply:
In that case the wiki needs to just be changed back too